packages/components/src/flex/flex-block/README.md

```diff
@@ -0,0 +1,15 @@
+# FlexBlock
+
+A layout component to contain items of a fixed width within `Flex`.
+
+## Usage
+
+See [`flex/README.md#usage`](../flex/README.md#usage) for how to use `FlexBlock`.
```

Comment:
We most likely need the absolute path so it also works at https://developer.wordpress.org/block-editor/reference-guides/components/:
```suggestion
See [`flex/README.md#usage`](/packages/componnents/src/flex/README.md#usage) for how to use `FlexBlock`.
```

Although I'm not sure if it's going to get properly replaced.